docs/deployment/docker.md

```diff
@@ -197,12 +197,13 @@ FROM node:20-bookworm-slim AS build
 
 # Set Python interpreter for `node-gyp` to use
 ENV PYTHON=/usr/bin/python3
+ENV NODE_OPTIONS="--max-old-space-size=8192"
```

Comment:
This isn't needed, if for some reason you need this due to your environment then feel free to add it toy our Dockerfile